dash/development/base_component.py

```diff
@@ -445,6 +589,27 @@ def generate_class(typename, props, description, namespace):
     result = scope[typename]
     return result
 
+def generate_class_r(typename, props, description, namespace):
```

Comment:
:zap: the generate_class for python is for dynamically loaded components, we don't do that in R.